tests that require bokeh are not running on CI

### Task Description

The GitHub workflow is not currently running any jobs that install bokeh, so tests that require it are not being run.

In particular, one test fails if bokeh is installed:
```Traceback (most recent call last):
  File "/mdao/u/swryan/benchmark/runner/repos/dymos/dymos/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py", line 209, in test_bokeh_plots
    self.assertTrue(html_file.exists(), msg=f'{html_file} does not exist!')
AssertionError: False is not true : reports/problem280/traj_results_report.html does not exist!
```

--- a/.github/workflows/dymos_tests_workflow.yml
+++ b/.github/workflows/dymos_tests_workflow.yml
@@ -35,7 +35,7 @@ jobs:
             PYOPTSPARSE: 'v2.9.3'
             SNOPT: 7.7
             OPENMDAO: 'latest'
-            OPTIONAL: '[test]'
+            OPTIONAL: '[all]'
             JAX: '0.3.24'
 
           # baseline versions except no pyoptsparse or SNOPT
@@ -238,15 +238,6 @@ jobs:
           else
             pip install openmdao==${{ matrix.OPENMDAO }}
           fi
-
-      - name: Install optional dependencies
-        if: env.RUN_BUILD && matrix.OPTIONAL == '[all]'
-        shell: bash -l {0}
-        run: |
-          echo "============================================================="
-          echo "Install additional packages for testing/coverage"
-          echo "============================================================="
-          pip install bokeh
 
       - name: Install Dymos
         if: env.RUN_BUILD

--- a/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
+++ b/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
@@ -194,9 +194,18 @@ def two_burn_orbit_raise_problem(transcription='gauss-lobatto', optimizer='SLSQP
 @use_tempdirs
 class TestExampleTwoBurnOrbitRaise(unittest.TestCase):
 
+    def setUp(self):
+        # We need to remove the TESTFLO_RUNNING environment variable for reports to be generated.
+        # The reports code checks to see if TESTFLO_RUNNING is set and will not do anything if set
+        self.testflo_running = os.environ.pop('TESTFLO_RUNNING', None)
+
     def tearDown(self):
         if os.path.isdir('plots'):
             shutil.rmtree('plots')
+
+        # restore what was there before running the test
+        if self.testflo_running is not None:
+            os.environ['TESTFLO_RUNNING'] = self.testflo_running
 
     @unittest.skipIf(not bokeh_available, 'bokeh unavailable')
     def test_bokeh_plots(self):

